receiver/elasticsearchreceiver/config_test.go

```diff
@@ -138,6 +140,7 @@ func TestValidateEndpoint(t *testing.T) {
 }
 
 func TestLoadConfig(t *testing.T) {
+	defaultTransport := http.DefaultTransport.(*http.Transport)
```

Comment:
One last thing, can you change this to match: https://github.com/open-telemetry/opentelemetry-collector/pull/11299/commits/aa03d7624950b58598e0fe1cbce27d408bf38424. Like that, we aren't affected if the global DefaultTransport is changed and define our own variables

Reply:
nice suggestion, changed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35509/commits/506af7876458b038f29d20e4e5850ca837000deb